AvaTax.TaxModule.Web/Scripts/blades/orders-synchronization-progress.js

```diff
@@ -0,0 +1,25 @@
+﻿angular.module('virtoCommerce.avataxModule')
```

Comment:
Display the json for already exists Avatax transaction for selected order and provide a link to this object in Avatax service.